packages/core/src/lib/utils/session.ts

```diff
@@ -0,0 +1,50 @@
+import { AdapterUser } from "../../adapters";
+import { JWTSessionError, SessionTokenError } from "../../errors";
+import { InternalOptions } from "../../types";
+import { SessionStore } from "./cookie";
```

Comment:
Makes sense, and I changed it to return regular `User` objects instead of `AdapterUser`. Regarding updating the session, I'm not sure if `getLoggedInUser` should do that. Reading from the session shouldn't necessarily update it, wdyt?